server/utilities/EmailModule.js

```diff
@@ -0,0 +1,48 @@
+import sendGridMail from '@sendgrid/mail';
+import dotenv from 'dotenv';
+
+dotenv.config();
+sendGridMail.setApiKey(process.env.SENDGRID_API_KEY);
+const { log } = console;
+
+/**
+ * Contains methods for handling user email services
+ *
+ * @exports
+ * @class EmailModule
+ */
+class EmailModule {
+  /**
+   * Send email message to a single user
+   *
+   * @static
+   * @async
+   * @name sendEmailToUser
+   * @param {object} receiver - Object contains "name" and "email" of receiver
+   * @param {string} subject - Subject of the email
+   * @param {string} content - Content of the email
+   * @returns {string} Information about whether or not the operation succeeds
+   * @memberof EmailModule
+   */
+  static sendEmailToUser(receiver, subject, content) {
+    const emailData = {
+      to: receiver,
+      from: 'Hello Books <hellobooks-care@hellobooks.com>',
+      subject,
+      html: content
+    };
+
+    sendGridMail
+      .send(emailData)
+      .then(() => {
```

Comment:
Great job @abdulfatai360 This looks great and very reusable throughout the application. 

However, could you please change the `then-ables` to use `async` and `await`  since this was the agreed-upon standard? thanks.

Reply:
@tolulope-od  I have edited that. This function is now written in `async...await` format. Thanks for pointing that out. I appreciate

Reply:
Thanks for that. You've done an awesome job!